packages/server/src/integrations/utils.ts

```diff
@@ -87,6 +87,10 @@ export function isExternalTable(tableId: string) {
   return tableId.includes(DocumentType.DATASOURCE)
 }
 
+export function isInternalTable(tableId: string) {
+  return !isExternalTable(tableId)
+}
```

Comment:
Not entirely sure if this is the best implementation of this, but I wanted to avoid writing `if (!isExternalTable(table))` in a conditional because I feel like it's the wrong intent. I wanted to make sure something was an internal table specifically.

Reply:
I think this is fine, I'm a fan of utility functions like this purely for it being easier to read what is going on!

Reply:
It feels like a small thing but I do think it matters that the intent of the code is clear. I don't want to find non-external tables, I want to find internal tables. That they're functionally equivalent doesn't matter imo.